apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/dashboard/_components/EvaluationStats.tsx

```diff
@@ -34,10 +37,14 @@ export default function EvaluationStats({
     useConnectedDocuments({ evaluation })
 
   useEffect(() => {
-    readMetadata({ prompt: evaluation.metadata.prompt }).then((metadata) => {
-      const metadataModel = (metadata.config['model'] as string) ?? 'Unknown'
-      setModel(metadataModel)
-    })
+    if (evaluation.metadataType === EvaluationMetadataType.LlmAsJudgeAdvanced) {
+      readMetadata({ prompt: evaluation.metadata.prompt }).then((metadata) => {
```

Comment:
can this promise fail? what should we do if it fails?

Reply:
Is should not fail. A fail here would mean something in the db is wrong, which should not happen and is covered by the create service.

I think in this case it is better to just let it fail if something happens, and get a Sentry notification